exonum-java-binding/core/src/main/java/com/exonum/binding/core/transaction/TransactionContext.java

```diff
@@ -54,14 +59,16 @@ default Prefixed getServiceData() {
 
   /**
    * Returns SHA-256 hash of the {@linkplain TransactionMessage transaction message} that
-   * carried the payload of the transaction.
-   * Each transaction message is uniquely identified by its hash; the messages are persisted
+   * carried the payload of the transaction; or a zero hash if no message corresponds to this
```

Comment:
Yes, it would make sense, since we are on Java 11, it has acquired some methods that make it easier to use than previously, where it added some friction.